mongodbatlas/cloud_provider_snapshot_restore_jobs_test.go

```diff
@@ -161,8 +163,10 @@ func TestCloudProviderSnapshotRestoreJobs_List(t *testing.T) {
 						Rel:  "http://mms.mongodb.com/group",
 					},
 				},
-				SnapshotID: "5b6211ff87d9d663c59d3feg",
-				Timestamp:  "2018-08-01T20:02:07Z",
+				SnapshotID:            "5b6211ff87d9d663c59d3feg",
+				Timestamp:             "2018-08-01T20:02:07Z",
+				OplogTs:               "2018-08-01T20:02:07Z",
+				PointInTimeUTCSeconds: 1583753751,
```

Comment:
Also fyi for later be sure when supporting in Terraform you note that deliveryType for doing a point in time restore job (PIT) would be pointInTime also note in the docs that you must specify either pointInTimeUTCSeconds or oplogTs, but not both.